aws/setContext.sh

```diff
@@ -217,32 +217,35 @@ if [[ -n "${PRODUCT}" ]]; then
     export APPSETTINGS_DIR="${CONFIG_DIR}/${PRODUCT}/appsettings"
     export CREDENTIALS_DIR="${INFRASTRUCTURE_DIR}/${PRODUCT}/credentials"
     
-    # slice level appsettings
-    if [[ (-n "${SLICE}") ]]; then
+    # deployment unit level appsettings
+    if [[ (-n "${DEPLOYMENT_UNIT}") ]]; then
         
-        # Confirm it is an application slice"
-        APPLICATION_SLICE_LIST=$(echo $(cat ${COMPOSITE_BLUEPRINT} | jq -r -f ${GENERATION_DIR}/listApplicationSlices.jq | dos2unix))
-        if [[ -n "$(echo ${APPLICATION_SLICE_LIST} | grep -i ${SLICE})" ]]; then
-            IS_APPLICATION_SLICE="true"
+        # Confirm it is an application deployment unit
+        APPLICATION_DEPLOYMENT_UNIT_LIST=$(echo $(cat ${COMPOSITE_BLUEPRINT} | jq -r -f ${GENERATION_DIR}/listApplicationDeploymentUnits.jq | dos2unix))
```

Comment:
APPLICATION_DEPLOYMENT_UNIT_LIST is empty.
this is the reason why generation of application template is not working.
I compared with master branch and didn't manage to understand why it is not being set properly after refactoring